.github/workflows/ci-tests.yml

```diff
@@ -36,11 +37,13 @@ jobs:
               - 'apollo-ios-pagination/**'
               - 'Tests/ApolloInternalTestHelpers/**'
               - 'apollo-ios/**'
+            tuist:
+              - '.tuist-version'
 
   tuist-generation:
-    runs-on: macos-13
+    runs-on: macos-14
     needs: [changes]
-    if: ${{ needs.changes.outputs.ios == 'true' || needs.changes.outputs.codegen  == 'true' || needs.changes.outputs.pagination  == 'true' }}
+    if: ${{ needs.changes.outputs.ios == 'true' || needs.changes.outputs.codegen  == 'true' || needs.changes.outputs.pagination  == 'true' || needs.changes.outputs.tuist == 'true' }}
```

Comment:
The tuist-generation job should run whenever we change the pinned Tuist version.